.github/workflows/ci.yml

```diff
@@ -26,7 +26,9 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Run Unit tests
-        run: sudo make test-coverage
+        run: |
+          sudo ln -sf `which go` `sudo which go` || true
+          sudo make test-coverage
```

Comment:
This issue was raised before in https://github.com/actions/setup-go/issues/104. `sudo` will change the Go version back to the default version, which is 1.15.

Reply:
Thx.

Reply:
Thanks!
This part has merged into the main branch, refer to https://github.com/dragonflyoss/Dragonfly2/pull/912
You can rebase main branch.